projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -200,6 +224,45 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDe
     });
   }
 
+  openSelectorDialog() {
```

Comment:
Small comment - that function can be probably private because it is not used outside component

Reply:
Changed to private, thanks